fenix/app/src/main/res/values/static_strings.xml

```diff
@@ -127,4 +127,21 @@
     <string name="translations_bottom_sheet_translating_in_progress">In progress</string>
     <string name="translations_bottom_sheet_translating_in_progress_content_description">Translating in Progress</string>
 
+
+    <string name="translation_option_bottom_sheet_title">Translation Options</string>
+    <string name="translation_option_bottom_sheet_always_translate">Always offer to translate</string>
+    <string name="translation_option_bottom_sheet_always_translate_in_language">Always translate %1$s</string>
+    <string name="translation_option_bottom_sheet_never_translate_in_language">Never translate %1$s</string>
+    <string name="translation_option_bottom_sheet_never_translate_site">Never translate this site</string>
+    <string name="translation_option_bottom_sheet_translation_settings">Translation settings</string>
+    <string name="translation_option_bottom_sheet_about_translations">About translations in %1$s</string>
+
+    <string name="translation_settings_fragment_toolbar_title">Translations</string>
+    <string name="translation_settings_fragment_translation_preference">Translation preferences</string>
+    <string name="translation_settings_fragment_offer_to_translate">Offer to translate when possible</string>
+    <string name="translation_settings_fragment_always_download">Always download languages in data saving mode</string>
+    <string name="translation_settings_fragment_automatic_translation">Automatic translation</string>
+    <string name="translation_settings_fragment_automatic_never_translate_site">Never translate this sites</string>
```

Comment:
Thanks for the change, very slight adjustment:
```suggestion
    <string name="translation_settings_fragment_automatic_never_translate_sites">Never translate these sites</string>
```

Reply:
Done